examples/jsm/math/Matrix2.js

```diff
@@ -0,0 +1,54 @@
+export class Matrix2 {
+
+	constructor( n11, n12, n21, n22 ) {
+
+		Matrix2.prototype.isMatrix2 = true;
+
+		this.elements = [
+			1, 0,
+			0, 1,
+		];
+
+		if ( n11 !== undefined ) {
+
+			this.set( n11, n12, n21, n22 );
+
+		}
+
+	}
+
+	identity() {
+
+		this.set(
+			1, 0,
+			0, 1,
+		);
+
+		return this;
+
+	}
+
+	fromArray( array, offset = 0 ) {
+
+		for ( let i = 0; i < 4; i ++ ) {
+
+			this.elements[ i ] = array[ i + offset ];
+
+		}
+
+		return this;
+
+	}
+
+	set( n11, n12, n21, n22 ) {
+
+		const te = this.elements;
+
+		te[ 0 ] = n11; te[ 1 ] = n21;
+		te[ 2 ] = n12; te[ 3 ] = n22;
```

Comment:
nit.
```js
te[ 0 ] = n11; te[ 2 ] = n12;
te[ 1 ] = n21; te[ 3 ] = n22;
```

I prefer the way `Matrix4` lays this out in human-readable form. In this PR, it is in transposed form. `Matrix3`, which I assume you used as a template, was [changed](https://github.com/mrdoob/three.js/pull/8228/files) for some reason, and I am tempted to change it back...

`.set()` is purposely in human-friendly, row-major order, while `.elements` is in the required column-major order.

Reply:
Sounds good to me - changed!